share/spice/coupons/coupons_triggers.txt

```diff
@@ -0,0 +1,140 @@
+aktionscode
+aktions code
+alekoodi
+alennus
+alennuskoodi
+alennuskuponki
+alennuslahjakortti
+alennusnumero
+arvoseteli
+bon achat
+bon code
+bon de reduction
+bon de réduction
+bon de remise
+bono
+bono descuento
+bon promocyjny
+bon rabatowy
+bon reduc
+bon reduction
+bon towarowy
+buoni sconto
+buono
+buono acquisto
+buono d'acquisto
+buono sconto
+cheque descuento
+cheque regalo
+code avantage
+code de reduction
+code de réduction
+code fidelite
+code privilege
+code promo
+code promotion
+code promotionnel
+code reduc
+code reduction
+code réduction
+code remise
+codice buono
+codice buono sconto
+codice coupon
+codice promo
+codice promozionale
+codice promozione
+codice sconto
+codice vantaggio
```

Comment:
My Italian colleague suggests we remove this one as it's a poor english -> italian translation